docs/docs/tutorials/get-started/1-build-basic-chatbot.md

```diff
@@ -87,39 +130,84 @@ def chatbot(state: State):
 # the node is used.
 graph_builder.add_node("chatbot", chatbot)
 ```
+:::
+
+:::js
+```typescript
+const chatbot = async (state: typeof State.State) => {
+  return { messages: [await llm.invoke(state.messages)] };
+};
+
+// The first argument is the unique node name
+// The second argument is the function or object that will be called whenever
+// the node is used.
+graphBuilder.addNode("chatbot", chatbot);
+```
+:::
 
 **Notice** how the `chatbot` node function takes the current `State` as input and returns a dictionary containing an updated `messages` list under the key "messages". This is the basic pattern for all LangGraph node functions.
 
+:::python
 The `add_messages` function in our `State` will append the LLM's response messages to whatever messages are already in the state.
+:::
+
+:::js
+The reducer function in our `State` will append the LLM's response messages to whatever messages are already in the state.
+:::
 
 ## 4. Add an `entry` point
 
 Add an `entry` point to tell the graph **where to start its work** each time it is run:
 
+:::python
 ```python
 graph_builder.add_edge(START, "chatbot")
 ```
+:::
+
+:::js
+```typescript
+graphBuilder.addEdge(START, "chatbot");
+```
+:::
 
```

Comment:
We should encourage chaining these, rather than invoking those methods separately